helm/templates/backend/backend.deployment.yaml

```diff
@@ -28,6 +28,9 @@ spec:
         - name: config
           configMap:
             name: {{ .Release.Name }}-backend
+        - name: logs
+          emptyDir: {}
+        {{ if .Values.loki.enabled }}
```

Comment:
Shouldn't the `name: logs` part also be conditional? If yes, it has to be changed for all occurrences.

Reply:
Yes in theory this also needs to be conditional but I thought this would get a bit messy. So you would prefer adjusting all loki related things (e.g., also adjust the guacamole container volume mounts or the guacd command)?

Reply:
In general: Yes. For the entrypoint, there is another change needed. Right now, we redirect stdour and stderr into the file - which doesn't print them in the kubernetes logs anymore. It would be better to still have it in the kubernetes logs. When this is implemented, we don't need a separation. 

More information: https://stackoverflow.com/a/692407